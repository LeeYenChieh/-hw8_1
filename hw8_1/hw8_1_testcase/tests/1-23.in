OdzRKJixyCgtivTQuNia8jEmBBtBj5r5mGBfyIciJzBGNX8Y408Ik cgE9lFOwpRC12HO1gwqDhWJIZ55Kx0hryTff1YcV
wy 84A
OdzRKJixyCgtivTQuNia8jEmBBtBj5r5mGBfyIciJzBGNX8Y408Ik display
aXzQ87SoNetuDFz2OTnbbTJ5AHgyCJiv1zepPFLuk lITjagh2j6jkq9jzOZs2Wh02
BVdr26BJwV2PLOFcawpV9ylVpIif IyxYpATqfhXGZ8G1VWusbeUqjVkB4xsb1VTldhUTCgvGQ4d
FNArkBN0CPu8Q4FgqBJeb36kIB vyhZLESolftQZVhW0rwUv
OdzRKJixyCgtivTQuNia8jEmBBtBj5r5mGBfyIciJzBGNX8Y408Ik add yQskkJ2izfKuv6rO2TRKQuN7RsbEndPddEpyDQF hEMtPeY8q2p7IMNKXMDvyCrdXIMBRwDLNvtj
wy display
GVRibQ52ZJ7dcCGgjumpC3RCNUdoCiZeqkQ1PMsrYrzj 6J38
