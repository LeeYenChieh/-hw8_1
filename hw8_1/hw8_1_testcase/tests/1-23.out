cgE9lFOwpRC12HO1gwqDhWJIZ55Kx0hryTff1YcV 
84A 
